docs/_docs/getting-started/dicom-pixels.md

```diff
@@ -335,7 +344,7 @@ that the fix is correct, and behavior might be changed in future versions of
 pydicom to raise an error. To disable the fix, you can do:
 
 ```python
-client.clean(fix_interpretation=False)
+client.clean_pixel_data(fix_interpretation=False)
```

Comment:
And here.

Could you please add an example to the docs of not using the client (if that is the purpose of the PR?) That will also help to explain the use case.

Reply:
https://github.com/pydicom/deid/pull/223/commits/b01a4530a84ab883f4d5fa666682f2970c172127